src/main/java/com/laserfiche/api/client/httphandlers/UsernamePasswordHandler.java

```diff
@@ -0,0 +1,74 @@
+package com.laserfiche.api.client.httphandlers;
```

Comment:
Minor question on the naming: is username password an OAuth flow? It looks similar to this OAuth flow: https://auth0.com/docs/get-started/authentication-and-authorization-flow/resource-owner-password-flow

Reply:
yes